src/_P090_MitsubishiHeatPump .ino

```diff
@@ -0,0 +1,698 @@
+//static const int RQST_PKT_STANDBY   = 5;
+
+static const int PACKET_LEN = 22;
+static const int PACKET_SENT_INTERVAL_MS = 1000;
+static const int PACKET_INFO_INTERVAL_MS = 2000;
+static const int PACKET_TYPE_DEFAULT = 99;
+
+static const int CONNECT_LEN = 8;
+static const byte CONNECT[CONNECT_LEN] = {0xfc, 0x5a, 0x01, 0x30, 0x02, 0xca, 0x01, 0xa8};
+static const int HEADER_LEN  = 8;
+static const byte HEADER[HEADER_LEN]  = {0xfc, 0x41, 0x01, 0x30, 0x10, 0x01, 0x00, 0x00};
+
+static const int INFOHEADER_LEN  = 5;
+static const byte INFOHEADER[INFOHEADER_LEN]  = {0xfc, 0x42, 0x01, 0x30, 0x10};
+
+static const int INFOMODE_LEN = 3;
+static const byte INFOMODE[INFOMODE_LEN] = {
+  0x02, // request a settings packet - RQST_PKT_SETTINGS
+  0x03, // request the current room temp - RQST_PKT_ROOM_TEMP
+//  0x04, // unknown
+//  0x05, // request the timers - RQST_PKT_TIMERS
+  0x06 // request status - RQST_PKT_STATUS
+//  0x09  // request standby mode (maybe?) RQST_PKT_STANDBY
+};
+
+static const int RCVD_PKT_FAIL            = 0;
+static const int RCVD_PKT_CONNECT_SUCCESS = 1;
+static const int RCVD_PKT_SETTINGS        = 2;
+static const int RCVD_PKT_ROOM_TEMP       = 3;
+static const int RCVD_PKT_UPDATE_SUCCESS  = 4;
+static const int RCVD_PKT_STATUS          = 5;
+//static const int RCVD_PKT_TIMER           = 6;
+
+static const byte CONTROL_PACKET_1[5] = {0x01,    0x02,  0x04,  0x08, 0x10};
+                               //{"POWER","MODE","TEMP","FAN","VANE"};
+static const byte CONTROL_PACKET_2[1] = {0x01};
+                               //{"WIDEVANE"};
+static const byte POWER[2]            = {0x00, 0x01};
+static const char* POWER_MAP[2]       = {"OFF", "ON"};
+static const byte MODE[5]             = {0x01,   0x02,  0x03, 0x07, 0x08};
+static const char* MODE_MAP[5]        = {"HEAT", "DRY", "COOL", "FAN", "AUTO"};
+static const byte TEMP[16]            = {0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f};
+static const int TEMP_MAP[16]         = {31, 30, 29, 28, 27, 26, 25, 24, 23, 22, 21, 20, 19, 18, 17, 16};
+static const byte FAN[6]              = {0x00,  0x01,   0x02, 0x03, 0x05, 0x06};
+static const char* FAN_MAP[6]         = {"AUTO", "QUIET", "1", "2", "3", "4"};
+static const byte VANE[7]             = {0x00,  0x01, 0x02, 0x03, 0x04, 0x05, 0x07};
+static const char* VANE_MAP[7]        = {"AUTO", "1", "2", "3", "4", "5", "SWING"};
+static const byte WIDEVANE[7]         = {0x01, 0x02, 0x03, 0x04, 0x05, 0x08, 0x0c};
+static const char* WIDEVANE_MAP[7]    = {"<<", "<",  "|",  ">",  ">>", "<>", "SWING"};
+static const byte ROOM_TEMP[32]       = {0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f,
+                                  0x10, 0x11, 0x12, 0x13, 0x14, 0x15, 0x16, 0x17, 0x18, 0x19, 0x1a, 0x1b, 0x1c, 0x1d, 0x1e, 0x1f};
+static const int ROOM_TEMP_MAP[32]    = {10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25,
```

Comment:
This is a lot of static allocations which are taking resources even when the plugin is not selected or enabled.
Please have a look at how static data can be stored in flash instead of in RAM (see [PROGMEM](https://arduino-esp8266.readthedocs.io/en/latest/PROGMEM.html) , which is similar to how the F() macro works for flash strings)
Also store all these in a struct which you only contruct and assign to a point when the task is initialized.
See for examples the GPS plugin and the P088 heatpump plugin.